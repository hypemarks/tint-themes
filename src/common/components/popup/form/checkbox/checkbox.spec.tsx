import { render } from '@testing-library/react';
import React from 'react';
import Checkbox from './checkbox';

describe('Test <Checkbox /> Component', () => {
  it('should render snapshot', () => {
    const wrapper = render(
      <Checkbox
        value={true}
        placeholder={'Click Me'}
        type={'Checkbox'}
        id={'id'}
        name={'check'}
        onBlur={jest.fn}
        onChange={jest.fn}
      />
    );
    expect(wrapper.asFragment()).toMatchSnapshot();
  });
});
