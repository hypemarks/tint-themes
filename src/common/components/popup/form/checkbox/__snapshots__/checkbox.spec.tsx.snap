// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test <Checkbox /> Component should render snapshot 1`] = `
<DocumentFragment>
  <input
    checked=""
    id="id"
    name="check"
    placeholder="Click Me"
    type="Checkbox"
  />
</DocumentFragment>
`;
